source/community/release_notes/teak/dev_op_release_notes.rst

```diff
@@ -306,6 +381,18 @@ Deprecations & Removals
   <https://github.com/openedx/brand-openedx/issues/23>`_, and follow the
   :ref:`Ulmo Design Tokens` page for more detail. Operators will be able to try
   out Design Tokens using the Teak Design Tokens branches (link TBD)
+- `[DEPR]: block_structure.storage_backing_for_cache in edx-platform <https://github.com/openedx/public-engineering/issues/32>`_ This is a simplification to how course content is cached. It should be invisible to all end users.
+- The flag ``ENABLE_BLAKE2B_HASHING`` was removed. blake2b hashing is now used for caching instead of the deprecated md4 hashing. After upgrading, it’s possible that performance could be degraded as the cache rebuilds. 
+- `[DEPR]: django-oauth2-provider (DOP) related tables <https://github.com/openedx/public-engineering/issues/82>`_
+
+   - For LMS and CMS, there is a new script to 
+     `clean up old DOP-related authentication tables. <https://github.com/edx/configuration/blob/master/util/drop_dop_tables/drop_dop_tables.py>`_
+   - If you have an old installation of the Open edX platform (Palm or later), you may have many outdated/unused 
+     authentication-related tables that can lead to confusion when looking at the database.
+   - The script is not related to Teak other than it now being available, and should be ok to run on any installation using Palm, Redwood, or Sumac.
+- `[DEPR]: Support for footer replacement via npm installing forked footers <https://github.com/openedx/frontend-component-footer/issues/459>`_
+   - It is possible to work around this breaking change by also exporting your forked ``Footer`` component as ``FooterSlot`` and your forked ``StudioFooter`` component as ``StudioFooterSlot``
+
```

Comment:
@kdmccormick - I think this is where to add an operator note about the `legacy_studio*` flags.